packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -899,14 +911,18 @@ export class CollaborationWindow {
 }
 
 /**
+ * Compares two numbers.
+ *
  * @internal
  */
-export const compareNumbers = (a: number, b: number) => a - b;
+export const compareNumbers = (a: number, b: number): number => a - b;
```

Comment:
(Not for this PR) I'm surprised we need to export these from the package, even if just `@internal`. Curious if anyone knows why these are currently being shared.

Reply:
It looks like they're used in partialLengths and a few tests, but I'm not sure if there's a specific reason to export them from here over defining them when needed. @Abe27342 thoughts?

Reply:
it does look like they are exported from the package, so the tag does nothing

Reply:
@anthony-murphy should it be removed then? I'm tracking some other cleanup with [AB#8760](https://dev.azure.com/fluidframework/internal/_workitems/edit/8760), so I can do it as part of that work if so.

Reply:
at minimum, the tag should be removed :-). We could also consider inlining them in the test cases we use them in if they're not strictly related (the comparisons are the idiomatic ones), but i'm kinda indifferent there.

Reply:
whatever you do here, follow-up pr is obviously fine. pre-existing.